### PR TITLE
DOCS-11257-aggregator-timebase-example-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Documented Time-Based Aggregator example
- Added Studio Steps and screenshots
- Reorganized content

### CODE REVIEW

File: aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
+= Time-Based Aggregator Example - Mule 4
+
```
Comment (kbostrom) [inline aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc:2]: Edit instructions: This is pretty similar to the previous PR you review yesterday, so you can just focus on the intro paragraph and the last paragraph after the Console logger message.

File: aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+The Aggregators module *Time based aggregator* scope enables you to aggregate elements until a pre-defined time period is completed. The following example shows how to configure this scope to aggregate HTTP request calls performed within `5` seconds. Then, the application transforms the calls via DataWeave into random ID numbers. When the aggregation reaches `5` seconds, an *Aggregator listener* source gets the list of elements and the application logs all the aggregated random ID numbers.
```
Comment (kbostrom) [inline aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc]:
```suggestion
The Aggregators module *Time based aggregator* scope enables you to aggregate elements within a predefined time limit. The following example shows how to configure this scope to aggregate HTTP request calls within `5` seconds, followed by showing how the application transforms the calls via DataWeave into random ID numbers. When the aggregation reaches `5` seconds, an *Aggregator listener* source gets the list of elements and the application logs all the aggregated random ID numbers.
```

File: aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+.Tize-Based Aggregator Flow Example
```
Comment (kbostrom) [inline aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc]:
```suggestion
.Time-Based Aggregator Flow Example
```

File: aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    "Source Name": "time-based",
+    "Source ID": "0.7095971146328517"
+  },
+  {
+    "Source Name": "time-based",
+    "Source ID": "0.6900116288202388"
+  }
+]
+```
+From the previous logger message, you can identify that for each HTTP request, the *Logger* component in the *Incremental aggregator* route logs the message:
```
Comment (kbostrom) [inline aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc]:
```suggestion
This example logger message shows that for each HTTP request, the *Logger* component in the *Incremental aggregator* route logs the message 
```

File: aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+`Doing incremental aggregation step.` After the `timeBasedAggregator` that is referenced by the *Aggregator listener* source completes the incremental aggregations within `5` seconds, the listener is triggered with a list of all the elements. The subsequent *Logger* component logs the message `Aggregation complete` and the second *Logger* component returns the aggregated payload elements, which are requests transformed into random ID numbers:
```
Comment (kbostrom) [inline aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc]:
```suggestion
`Doing incremental aggregation step.` After the `timeBasedAggregator` value that is referenced by the *Aggregator listener* source completes the incremental aggregations within `5` seconds, the listener is triggered with a list of all the elements. The subsequent *Logger* component logs the message `Aggregation complete`, and the second *Logger* component returns the aggregated payload elements, which are requests transformed into random ID numbers:
```

**Review by kbostrom — DISMISSED** [review]
CE

**Review by giss-ignacio — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**